Userland/Libraries/LibWeb/HTML/HTMLScriptElement.cpp

```diff
@@ -50,6 +52,8 @@ void HTMLScriptElement::execute_script()
     // 1. Let document be scriptElement's node document.
     JS::NonnullGCPtr<DOM::Document> node_document = document();
 
+    // FIXME: Update the steps to match the newer spec.
```

Comment:
(Maybe add a link to the new spec to make this more actionable?)

Reply:
This applies to the whole function which is marked with a spec url already. I adjusted the working slightly to hopefully make the intention more clear.

```diff
-FIXME: Update the steps to match the newer spec.
+FIXME: Update the steps to match current spec.
```

Reply:
~Ok I might be misunderstanding / know too much in this case.
Is this comments about the spec currently on whatwg? Because the comments (and code) seems to match the spec at the link.
Or do you mean the new PR which rewrites the module code?~ Oh just saw this was merged nvm yeah this is good